clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -261,6 +273,49 @@ option_init(int argc, const char *argv[])
             return false;
         }
     }
+
+    /* Because Intel PT doesn't save instruction bytes or memory contents, the converter
+     * needs the instruction bytes for each IPs (Instruction Pointers) to decode PT.
+     * drpt2trace supports two modes to convert:
+     * (1) Sideband Mode: the user must provide sideband data and parameters. In this
+     * mode, the converter uses sideband decoders to simulate image switches during the
+     * conversation. We can use this mode to convert the traces that the instruction bytes
+     * located in multiple images.
+     * (2) Elf Mode: the user needs to provide an elf image for the PT. This mode is for
+     * cases where the PT traces' instruction bytes belong to one image. So we can use
+     * this mode to convert the kernel trace and the short-term trace.
```

Comment:
short-term user trace where it's likely that we'll not have an image switch.

Reply:
Done.